lib/scrivener/paginater/ecto/query.ex

```diff
@@ -61,12 +65,22 @@ defimpl Scrivener.Paginater, for: Ecto.Query do
     |> exclude(:select)
   end
 
-  defp count(query) do
+  defp count(query, opts \\ []) do
+    prefix = get_prefix(opts)
+
     query
+    |> Map.put(:prefix, prefix)
     |> subquery
     |> select(count("*"))
   end
 
+  defp get_prefix(opts \\ []) do
+    case Keyword.fetch(opts, :prefix) do
```

Comment:
If you're looking to return `nil` in the case `:prefix` isn't found, why use `fetch` rather than `opts[:prefix]`?